lib/quill_delta.dart

```diff
@@ -442,6 +467,80 @@ class Delta {
     return result..trim();
   }
 
+  /// Returns a new lazy Iterable with elements that are created by calling f
```

Comment:
Probably need to rename `f` to `predicate` to match the method definition.

Reply:
Renamed argument to `f`